iceoryx_utils/test/moduletests/test_cxx_variant_queue.cpp

```diff
@@ -120,6 +130,41 @@ TEST_F(VariantQueue_test, handlesOverflow)
     });
 }
 
+TEST_F(VariantQueue_test, blocksOnOverflow)
+{
+    PerformTestForQueueTypes([](uint64_t typeID) {
+        VariantQueueTypes type = static_cast<VariantQueueTypes>(typeID);
+        if (!(type == VariantQueueTypes::BlockingFiFo_MultiProducerSingleConsumer
+              || type == VariantQueueTypes::BlockingFiFo_SingleProducerSingleConsumer))
+        {
+            return;
+        }
+
+        VariantQueue<int, 2> sut(type);
+
+        for (uint64_t i = 0U; i < sut.capacity(); ++i)
+        {
+            sut.push(24123 + i);
+        }
+
+        std::atomic_uint64_t counter{0U};
```

Comment:
/home/runner/work/iceoryx/iceoryx/iceoryx_utils/test/moduletests/test_cxx_variant_queue.cpp:150:9: error: ‘atomic_uint64_t’ is not a member of ‘std’
         std::atomic_uint64_t counter{0U};